util/chunk/chunk_util_test.go

```diff
@@ -61,7 +61,7 @@ func TestCopySelectedJoinRows(t *testing.T) {
 	}
 	// batch copy
```

Comment:
We should add a test case covering this situation here.

Reply:
done